src/main/kotlin/no/skatteetaten/aurora/boober/controller/internal/Payloads.kt

```diff
@@ -1,10 +1,9 @@
 package no.skatteetaten.aurora.boober.controller.internal
 
-import com.fasterxml.jackson.databind.JsonNode
 import no.skatteetaten.aurora.boober.model.ApplicationId
 import no.skatteetaten.aurora.boober.model.AuroraConfigFile
 
-typealias JsonDataFiles = Map<String, JsonNode>
+typealias JsonDataFiles = Map<String, String>
```

Comment:
I guess this is not exclusively Json anymore? Maybe a different name?

Reply:
I think we can just remove the alias alltogether. It is just a map now.